teacher.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    super(age, parent_permission: parent_permission)
+    @specialization = specialization
+    @name = name
```

Comment:
- [ ] It would be better to pass `name` to  `super()` and delete it from this class that will keep you from dry (don't repeat yourself )